README.md

```diff
@@ -256,7 +256,12 @@ VoodooPS2Controller.kext
 WhateverGreen.kext
 ```
 
->**Note**: If you plan to install macOS Big Sur or older (11≥), your Bluetooth kexts will be different. Read the Dortania guide to find out what you need.
+>**Note**: 
+> - If you plan to install macOS Big Sur or older (11≥), your Bluetooth kexts will be different. Read the Dortania guide to find out what you need.
+> - You might need to clean-up PlugIns in `VoodooPS2Controller.kext` except of `VoodooPS2Keyboard.kext`. `VoodooI2C.kext` and `VoodooI2CELAN.kext`from separate repo are still needed if you go down this path.
+> - Don't add blindly `IntelBluetoothInjector.kext` from the `IntelBluetoothFirmware` repo, it conflicts with `EmeraldSDHC` and external USB.
+> - Most of the USB seems to work without USB mapping nowadays (tested with C433), hence `UTBMap.kext` & `USBToolBox.kext` are optional.
```

Comment:
Correct, but it is good practice to do so.

Reply:
👍